actinia-alpine/Dockerfile

```diff
@@ -126,6 +136,13 @@ RUN mkdir -p /actinia_core/grassdb && \
 VOLUME /grassdb
 WORKDIR /src/actinia_core
 
+# install GDAL GRASS plugin (required for i.sentinel1.pyrosargeocode)
+RUN mkdir -p /usr/lib/gdalplugins
+RUN git clone https://github.com/OSGeo/gdal-grass.git
+RUN cd gdal-grass && ./configure --with-gdal=/usr/bin/gdal-config --with-grass=/usr/local/grass83 \
+&& make && make install
+RUN cd .. && rm -rf gdal-grass
+
```

Comment:
I would prefer these lines to be in the `Dockerfile_alpine_dependencies`. The goal is to have as much there with a scheduled build which can take as long as it needs and then to have as little things installed here as possible to make the build as quick as possible.

Reply:
If I understand correctly, the GRASS installation comes from `FROM osgeo/grass-gis:releasebranch_8_3-alpine as grass`. This means that `Dockerfile_alpine_dependencies` does not have any GRASS installed, but the `gdal-grass` plugin needs to be installed with an existing GRASS and GDAL installation: 
```
RUN cd gdal-grass && ./configure --with-gdal=/usr/bin/gdal-config --with-grass=/usr/local/grass83 \
&& make && make install
```
So moving the gdal-grass lines to `Dockerfile_alpine_dependencies` yields
```
0.816 configure: error: --with-grass=/usr/local/grass83 requested, but libraries not found!
```

Reply:
You are right :innocent: 
How long does it take to compile?
The multistage approach tries to have the best of everything - fast build times and most recent code. If it takes too long, we might need to come up with a different solution.

Reply:
This bit 
```
RUN mkdir -p /usr/lib/gdalplugins
RUN git clone https://github.com/OSGeo/gdal-grass.git
RUN cd gdal-grass && ./configure --with-gdal=/usr/bin/gdal-config --with-grass=/usr/local/grass83 \
&& make && make install
RUN cd .. && rm -rf gdal-grass
```
takes ~ 10 seconds in the docker build command. What really takes time is the installation of GRASS addons (https://github.com/actinia-org/actinia-docker/blob/main/actinia-alpine/Dockerfile#L57) with 5-10 minutes in my local tests. All other lines only take a few seconds max

Reply:
Ok then fine for me!